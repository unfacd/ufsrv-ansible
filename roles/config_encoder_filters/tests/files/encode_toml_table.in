table-1:
  key1: some string
  key2: 123

dog:
  tater.man:
    type: pug

x:
  y:
    aaa: bbb
    z:
      w:
        name: deep table
