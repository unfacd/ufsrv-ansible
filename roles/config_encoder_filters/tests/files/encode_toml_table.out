[dog."tater.man"]
type = "pug"

[table-1]
key1 = "some string"
key2 = 123

[x.y]
aaa = "bbb"

[x.y.z.w]
name = "deep table"
